<launch>

    <include file="$(find projeto_semear)/launch/pid.launch"/>
    <include file="$(find projeto_semear)/launch/simuladorInterface.launch"/>

    <node name="goToFirstPose" pkg="projeto_semear" type="goToFirstPose.py" respawn="false" output="screen"/>   
    <node name="goToContainer" pkg="projeto_semear" type="goToContainer.py" respawn="false" output="screen"/>   
    <node name="main" pkg="projeto_semear" type="main.py" respawn="false" output="screen"/>   

</launch>   
